cmd/habitat/main.go

```diff
@@ -46,6 +49,12 @@ func main() {
 	reverseProxy := proxy.NewServer()
 	go reverseProxy.Start(fmt.Sprintf("%s:%s", ReverseProxyHost, ReverseProxyPort))
 
+	// Start data proxy
```

Comment:
Maybe more of a long term unification of ideas, but it would be cool to have the data proxy be built into our existing reverse proxy. We could add a new implementation of the `proxy.Rule` interface called `DataProxy`. The main advantage would be opening one less port + keeping things less confusing (one less thing named proxy). I don't think this will be overloading the existing reverse proxy, because the `Rule` interface gives us a lot of flexibility

Reply:
Yeah, i agree with that direction. Down to do this refactor sooner honestly since there's less code right now